### ISSUE TITLE
assumes build/contracts

### ISSUE THREAD

**Issue by mrinalwadhwa** [sc-forks/solidity-coverage#242]
Truffle allows setting a custom `contracts_build_directory`
http://truffleframework.com/docs/advanced/configuration#contracts_build_directory

If that is set `solidity-coverage` prints this during runs:
```
ls: no such file or directory: ./coverageEnv/build/contracts/*.json
```

**Comment by cgewecke** [comment]
@mrinalwadhwa Ah yes - that makes sense. Could you give an example of your config so that when we fix this we have test case to validate it with?

**Comment by mrinalwadhwa** [comment]
Sure @cgewecke, thank you for your work on `solidity-coverage`.

My `truffle-config.js` looks like this
```
var path = require('path');

module.exports = {
  contracts_build_directory: path.join(__dirname, './dist/contracts/'),
  networks: {
    development: {
      host: 'localhost',
      port: 8545,
      network_id: "*"
    },
    coverage: {
      host: 'localhost',
      network_id: '*',
      port: 8555,
      gas: 0xfffffffffff,
      gasPrice: 0x01,
    }
  }
};
```

Had to account for this truffle issue 
https://github.com/trufflesuite/truffle/issues/862#issuecomment-377952233

`.solcover.js` looks like this.

```
module.exports = {
  norpc: true
}
```
I'm starting `testrpc-sc` in a script before running `solidity-coverage`

```
node_modules/.bin/testrpc-sc" --gasLimit 0xfffffffffff --port 8555 --account="0x..." > /dev/null &
```

**Comment by cgewecke** [comment]
Excellent! Thank you.

**Comment by cgewecke** [comment]
This should now be configurable *in .solcover.js*  thanks to #272. Published with `0.5.7`. Closing but please ping (anyone) if this doesn't resolve the problem.
